opentelemetry-instrumentation/src/opentelemetry/instrumentation/_semconv.py

```diff
@@ -367,10 +391,11 @@ def _set_status(
     status_code: int,
     status_code_str: str,
     server_span: bool = True,
-    sem_conv_opt_in_mode: _HTTPStabilityMode = _HTTPStabilityMode.DEFAULT,
+    sem_conv_opt_in_mode: _StabilityMode = _StabilityMode.DEFAULT,
 ):
     if status_code < 0:
-        metrics_attributes[ERROR_TYPE] = status_code_str
+        if _report_new(sem_conv_opt_in_mode):
```

Comment:
Not related change, but when working on this noticed we miss this if. ERROR_TYPE is required only for stable semconv.